eland/operations.py

```diff
@@ -40,6 +40,16 @@
     SizeTask,
 )
 
+with warnings.catch_warnings() as w:
```

Comment:
You can remove the `w` as its not used

Reply:
Done!

Reply:
Oh I meant adding another capture_warning() within the test case and then doing `assert w == []` to show that your fix is working and not throwing warnings

Reply:
Done! Sorry for the mistake.

Reply:
No problem! I wasn't clear enough :)